sys/include/fs/littlefs2_fs.h

```diff
@@ -95,7 +95,7 @@ typedef struct {
      * total number of block is defined in @p config.
      * if set to 0, the total number of sectors from the mtd is used */
     uint32_t base_addr;
-    uint8_t sectors_per_block;  /**< number of sectors per block */
+    uint16_t sectors_per_block; /**< number of sectors per block */
```

Comment:
I made this a `uint16_t` because I chose the maximum of `CONFIG_LITTLEFS2_MIN_BLOCK_SIZE_EXP` to be 2^15 in Kconfig.